app/src/main/java/org/simple/clinic/patient/businessid/Identifier.kt

```diff
@@ -79,7 +87,19 @@ data class Identifier(
     object EthiopiaMedicalRecordNumber : IdentifierType()
 
     @Parcelize
-    object IndiaNationalHealthId : IdentifierType()
+    object IndiaNationalHealthId : IdentifierType() {
+
+      @IgnoredOnParcel
+      const val ENTERED_CODE_LENGTH = 14
+
+      fun enteredCodeLength(identifier: Identifier): String {
```

Comment:
I don't think we need this. It was added for short code, because we need to get short code from the UUID. But since we are directly saving the 14 digit NHID we don't have to do this here.